lib/Query.php

```diff
@@ -799,8 +799,10 @@ public function __call($method, $args)
 
             // Methods on Collection
         } elseif (method_exists('\\Spot\\Entity\\Collection', $method)) {
+            if(isset($args[0]))
```

Comment:
All `if` statements need to use curly braces and proper indentation. Fix this and I'll merge it :)